server/src/main/java/org/opensearch/index/mapper/DocumentMapper.java

```diff
@@ -236,7 +239,7 @@ public MappingLookup mappers() {
         return this.fieldMappers;
     }
 
-    public FieldTypeLookup fieldTypes() {
+    FieldTypeLookup fieldTypes() {
```

Comment:
The `FieldTypeLookup` type is package protected, there is no reasons for this method to stay public since the caller won't be able to access the return value.